File: 2A_1/python/funkcje01.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  funkcje01.py
#  

def sumuj(l1, l2):
    """
    Funkcja sumuje dwie podane liczby i drukuje wynik.
    """
    suma = l1 + l2
    print("Suma:", suma)


def odejmij(l1, l2):
    """
    Funkcja odejmuje dwie podane liczby i drukuje wynik.
    """
    roznica = l1 - l2
    print("Różnica:", roznica)


def pomnoz(l1, l2):
    """
    Funkcja mnozy dwie liczby i zwraca wynik.
    """
    iloczyn = l1 * l2
    return iloczyn


def main(args):
    liczba1 = int(input("Podaj liczbę: "))
    liczba2 = int(input("Podaj liczbę: "))
    
    # sumuj(liczba1, liczba2)  # wywołanie funkcji
    # odejmij(liczba1, liczba2)  # wywołanie funkcji
    
    print("Iloczyn:", pomnoz(liczba1, liczba2))
    # wynik = pomnoz(liczba1, liczba2)
    # print("Iloczyn:", wynik)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: 2A_1/cpp/funkcje01.cpp
/*
 * funkcje01.cpp
 * 
 */


#include <iostream>

using namespace std;

void sumuj(int l1, int l2) {
	int suma = l1 + l2;
	cout << "Suma: " << suma << endl;
}

void odejmij(int l1, int l2) {
	int roznica = l1 - l2;
	cout << "Różnica: " << roznica << endl;
}

int pomnoz(int l1, int l2) {
	int iloczyn = l1 * l2;
	return iloczyn;
}

int main(int argc, char **argv)
{
	int liczba1, liczba2;
	cout << "Podaj liczby: ";
	cin >> liczba1;
	cin >> liczba2;
	
	// sumuj(liczba1, liczba2);
	// odejmij(liczba1, liczba2);
	
	cout << "Iloczyn: " << pomnoz(liczba1, liczba2) << endl;
	// int wynik = pomnoz(liczba1, liczba2);
	// cout << "Iloczyn: " << wynik << endl;
	
	return 0;
}

